### DISCUSSION TITLE
The tic80 site is not working

### DISCUSSION THREAD

**Discussion by riko0800r** [nesbox/TIC-80#2699] [Q&A]
the tic80 website is not working, i have a game jam.. so make it fast
![Captura de tela 2024-10-07 113721](https://github.com/user-attachments/assets/59c63e78-5cbd-4c1e-acb3-7a14631a92c7)
![Captura de tela 2024-10-07 122438](https://github.com/user-attachments/assets/de9afeb5-97f0-45e4-97c9-f0daa4f43bc9)

**Comment by Miguel-hrvs** [comment]
@nesbox if you have some time to look into this, please help :)

**Reply by nesbox** [reply]
hmm, trying to understand...

**Comment by nesbox** [comment]
Now everything works, thanks for the warning.

**Reply by Miguel-hrvs** [reply]
Thanks, all great now :)
